acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/accountManagement/AccountManagementAcceptanceTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.tests.accountManagement;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import tech.pegasys.ethsigner.tests.AcceptanceTestBase;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.junit.Test;
+
+public class AccountManagementAcceptanceTest extends AcceptanceTestBase {
+
+  @Test
+  public void ethSignerAccountsReturnsAccountForPrivKey() throws Exception {
+    List<String> accounts = ethSigner().accounts().list();
+    assertThat(accounts.size()).isEqualTo(1);
+    assertThat(ethNode().accounts().balance(accounts.get(0))).isNotNull();
+  }
+
+  @Test
+  public void ethAccountsReturnsNoAccounts() throws IOException {
```

Comment:
Is this Exception actually needed?

Reply:
needed

Reply:
After fixing the Accounts and Transactions classes this is no longer needed!